### PR TITLE
How to Contribute to ETC: Community Volunteering

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is part of the series "How to Contribute to ETC", and this one is about volunteering.

Chinese version done.

This one will be published on 2023-03-21.

### CODE REVIEW

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+- Frequently asked questions
+- The “Why Classic?” section
+- The Knowledge base
+- A Services page
+- A Community Page
+- A Network page
+- A Mining page
+- A Development page
+- All other static pages (About, Sitemap, etc.)
+
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:130]: Line 130: Addition
```suggestion

For more details on how to contribute items for inclusion on the ETC website see an earlier article from this series entitled ["How to Contribute to ETC: The Community Website"](https://www.ethereumclassic.org/blog/2023-01-03-how-to-contribute-to-etc-community-website).

```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+## Research, Writing, and Producing Videos
+
+![ETC contributors.](./6.png)
+
+One of the most valuable ways of volunteering for ETC is by using your talents and knowledge to advance the network. There are many contributors who do research, write, and produce videos regularly for Ethereum Classic.
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:135]: Line 135: Revise word order; Correct usage
```suggestion
One of the most valuable ways of volunteering for ETC is by using your talents and knowledge to advance the network. There are many contributors who regularly do research, write, and produce videos about Ethereum Classic.
```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+A great way of doing this is by starting your own ETC YouTube channel and/or blog. Examples of these are:
+
+- Etherplan: https://etherplan.com
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:139]: Line 139: Embed link
```suggestion
- [Etherplan](https://etherplan.com)
```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+- PatientMoney: https://www.youtube.com/@PatientMoney
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:140]: Line 140: Embed link
```suggestion
- [PatientMoney](https://www.youtube.com/@PatientMoney)
```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+- SavageProfits: https://www.youtube.com/@SavageProfits
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:141]: Line 141: Embed link
```suggestion
- [SavageProfits](https://www.youtube.com/@SavageProfits)
```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+- StonkInsights: https://www.youtube.com/@stonkinsights
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:142]: Line 142: Embed link
```suggestion
- [StonkInsights](https://www.youtube.com/@stonkinsights)
```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+- Ethereum Classic Updates: https://www.youtube.com/@ETCupdates
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:143]: Line 143: Embed link
```suggestion
- [Ethereum Classic Updates](https://www.youtube.com/@ETCupdates)
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]
Changes requested, although most are simply suggestions

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+- Following on Twitter: [@eth_classic](https://twitter.com/eth_classic) and [@etc_network](https://twitter.com/etc_network)
+- ETC Telegram: [@ethclassic_army](https://t.me/ethclassic_army) and [@KimiCapital](https://t.me/KimiCapital)
+- ETC Cooperative Discord: https://discord.gg/5wDyd6u6pU
+- Discussions on GitHub: https://github.com/ethereumclassic 
+- Reddit: https://reddit.com/r/EthereumClassic
+- WeChat in Chinese: https://etccooperative.org/nanli777.jpg (there you will find the QR code for the WeChat channel)
+
+## Moderating Social Channels
+
+![ECIP editors.](./3.png)
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:68]: Liine 68: Move graphic to ECIP section
```suggestion
```

File: content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+- Etc_Network account: 
+
+https://github.com/ethereumclassic/tweets-etc_network
+
+Anyone with tweet suggestions may submit them by following the instructions found in the repositories' README.ml file. Tweets are filtered and managed by volunteer ETC admins who take care that the content adheres to Twitter's rules and ETC’s principles.
+
+## Debating Proposals of the ECIP Process
+
+As ETC is an open source project, any changes to its protocol or software must go through a public process called the Ethereum Classic Improvement Proposal (ECIP) process.
```
Comment (TheCrowbill) [inline content/blog/2023-03-21-how-to-contribute-to-etc-community-volunteering/index.md:98]: Line 98: Insert graphic
```suggestion
![ECIP editors.](./3.png)

As ETC is an open source project, any changes to its protocol or software must go through a public process called the Ethereum Classic Improvement Proposal (ECIP) process.
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]

**Review by gitr0n1n — APPROVED** [review]

**Review by brolal — APPROVED** [review]

**Review by chris-mercer — APPROVED** [review]

**Pull request merged**